verification/verify_test.go

```diff
@@ -22,6 +25,10 @@ func (sv *stubVerifier) Verify(params *Params) (*Results, error) {
 	return sv.results, sv.err
 }
 
+func (sv *SegmentVerifier) VerifyNoSig(params *Params) (*Params, error) {
```

Comment:
Hm feels a bit weird to me to modify the API of `SegmentVerifier` here even if it is only for testing. Another option could be to specify `sigVerifyFunc` as a field on `SegmentVerifier` that can use `lpcrypto.VerifySig()` by default in `NewSegmentVerifier()` and overridden with a stub implementation within the test. WDYT?

Reply:
resolved by d3e4afb5f4f5f4a90b0982d7e18ad55a5af06db2